### ISSUE TITLE
Review of and Team Walkthrough of ArcGIS tool

### ISSUE THREAD

**Issue by snooravi** [hackforla/access-the-data#85]
### Overview
We will need to understand how to use the ArcGIS tool in order to structure and understand how data is visualized in this tool. 

### Additional Information
- #128  finalize story for 311 workshop and be ready to identify requirements for visualization tools 

### Action Items

- [ ] Play around with [ArcGIS](https://www.esri.com/en-us/arcgis/products/create-account) with LA data
- [ ] Give the team a walkthrough of the tool

**Comment by MalakH21** [comment]
Here's an example of a visualization that can be created using layering + address adjustments with ArcGIS
<img width="782" alt="Screen Shot 2021-10-27 at 7 52 19 PM" src="https://user-images.githubusercontent.com/40288906/140842471-cdf8be72-8ab8-4443-b098-21534003e894.png">

**Comment by mcmorgan27** [comment]
Maybe we could discuss this next week?  What does layering + address adjustments mean?  What are the layers you're using?  I see an earlier comment about "find easy way to go from address -> lat, long" ... are you using an ESRI geocoder?  I have good luck using [nominatim](https://nominatim.org/) for geocoding. 

I'd like to know more.

**Comment by MalakH21** [comment]
@mcmorgan27 yes! layering in reference to ArcGIS, is how you add geographical data within the tool. Each "layer" is the volume of service requests by a service request type within a NC, taken from https://hackforla.github.io/311-report/ the Alpha tool. Honestly, I was stumbling around through ArcGIS, so if you'd like to test out the tool with the Alpha tool data, please do!

The lat, long conversion is for the Palladio tool since it doesn't accept addresses. But, yes, it seems that nominatim would be useful for both Palladio and ArcGIS, since the Alpha tool provides incomplete addresses namely missing city name.

**Comment by MalakH21** [comment]
next @mcmorgan27 @MalakH21 to discuss geocoding, alternative visualization methods

**Comment by mcmorgan27** [comment]
@MalakH21 do you have some candidate times we might meet?  I have questions ...

**Comment by snooravi** [comment]
@MalakH21 Any update on this issue?

**Comment by MalakH21** [comment]
@MalakH21 @mcmorgan27 to meet on Nov. 23rd to review potential usage plans for ArcGIS and/or alternative software, 
will update the issue after that meeting

**Comment by MalakH21** [comment]
moving to ice box - will revisit once guiding narrative is determined to better guide choice around visualization software + attaining licensing for said software if needed
